ui/imports/utils/Constants.qml

```diff
@@ -329,22 +329,26 @@ QtObject {
     }
 
     readonly property QtObject settingsSubsection: QtObject {
-        property int profile: 0
-        property int contacts: 1
-        property int ensUsernames: 2
-        property int messaging: 3
-        property int wallet: 4
-        property int appearance: 5
-        property int language: 6
-        property int notifications: 7
-        property int syncingSettings: 8
-        property int browserSettings: 9
-        property int advanced: 10
-        property int about: 11
-        property int communitiesSettings: 12
-        property int keycard: 13
-        property int signout: 14
-        property int backUpSeed: 15
+        readonly property int profile: 0
+        readonly property int contacts: 1
+        readonly property int ensUsernames: 2
+        readonly property int messaging: 3
+        readonly property int wallet: 4
+        readonly property int appearance: 5
+        readonly property int language: 6
+        readonly property int notifications: 7
+        readonly property int syncingSettings: 8
+        readonly property int browserSettings: 9
+        readonly property int advanced: 10
+        readonly property int about: 11
+        readonly property int communitiesSettings: 12
+        readonly property int keycard: 13
+        readonly property int about_terms: 14
```

Comment:
why not `aboutTerms`?

Reply:
Yeah... a bit undecided on that; I can do if you really really want :)

Reply:
Ah I recall now the reason; I wanted to underline (pun intended :) it is a subpage, unlike a regular page (as in `communitiesSettings`)

Reply:
Ok, I understand, but for the reader the idea behind is not clear at all. Even for you as a reader after some time ;) So maybe it's worth to add some comment what's up here. Otherwise someone will "align" go camel case in the future.

Reply:
Alright, fixed; added a comment there